db/seeds.rb

```diff
@@ -21,33 +21,38 @@
 GENERATE_TAGS_COUNT = 30
 GENERATE_POSTS_COUNT = 5
 GENERATE_SUBSCRIPTIONS_COUNT = 10
+GENERATE_CONVERSATIONS_COUNT = 6
 GENERATE_PROFILE_COUNT = 6
+USER_SUBSCRIPTIONS_COUNT = 10
 
 users = FactoryBot.create_list(:user, NUMBER_OF_USERS)
 users.each do |user|
   FactoryBot.create_list(:post, GENERATE_POSTS_COUNT, user: user)
   FactoryBot.create_list(:post_with_pictures_and_likes, GENERATE_POSTS_COUNT, user: user)
 end
 
+USER_SUBSCRIPTIONS_COUNT.times do
+  user1 = users.sample
+  user2 = users.sample
+  loop do
+    user2 = users.sample
+    break if user2 == user1
+  end
```

Comment:
```
User.all.each_slice(2) do |subscriber, subscription|
  UserSubscription.create(subscriber_id: subscriber.id, subscription_id: subscription.id)
end
```
Probably we can make it way simpler. Let's have just 1 subscriber and 1 subscription

Reply:
I have a proposal:
```
users.each do |user|
  subscribers = users
  subscribers.delete(user)
  rand(0..(NUMBER_OF_USERS-1)).times do
    FactoryBot.create(:user_subscription, subscriber: user, subscription: subscribers.delete(subscribers.sample)) 
    # not sure if it's possible to do the same with create_list
  end
end
```